src/messages.json

```diff
@@ -0,0 +1,10 @@
+{
+  "pagination.prev": "Go to previous page",
+  "pagination.next": "Go to next page",
+  "pin.toolbar": "Pin the controls toolbar",
+  "fullscreen.toggle": "Go/Exit fullscreen",
```

Comment:
this needs one message for each action: `Enter fullscreen` `Exit fullscreen`

Reply:
I added one message because only one button is present unlike the pagination ones.

Reply:
yeah but is a different action

Reply:
See how the `youtube` player handles tooltips for example :v:

Reply:
I see what you mean. The logic also will have to be tweaked a little to accommodate this, let me try.

Reply:
Done 🙂